### ISSUE TITLE
Woocommerce Module Deprecated Function

### ISSUE THREAD

**Issue by kennedy686** [qtranslate/qtranslate-xt#686]
Notice: Το woocommerce_get_product_attributes is deprecated.
You have to change line 69 in qwc-front.php 
add_filter( 'woocommerce_get_product_attributes', 'qwc_get_product_attributes', 5 );
with
add_filter( 'woocommerce_product_get_attributes', 'qwc_get_product_attributes', 5 );

**Comment by herrvigg** [comment]
Thanks! But... this change is very, very old!! They deprecated it in WC 3.0.0 if the version is correct in `class-wc-deprecated-filter-hooks`. But this sends us back to... 2017! This surprises me since many people have used this meanwhile. Could this explain some of the current issues? Maybe! I'd need some WC users to help me find the related issues, for example looking through: https://github.com/qtranslate/qtranslate-xt/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+woocommerce

I also found that `woocommerce_order_tax_totals` was replaced with `woocommerce_order_get_tax_totals` but that's more of a minor thing.

I'm also pretty sure there are many filters that can be added to the module for better support but i can't really say what matters. Would be good to have more developers using WC involved.
